wasm/jit/mmap_darwin.go

```diff
@@ -1,5 +1,24 @@
+//go:build darwin
+// +build darwin
+
 package jit
 
 import "syscall"
 
 const mmapFlags = syscall.MAP_ANON
+
+// Copy the code into the executable region
+// and returns the byte slice of the region.
+func mmapCodeSegment(code []byte) ([]byte, error) {
+	mmapFunc, err := syscall.Mmap(
+		-1,
+		0,
+		len(code),
+		syscall.PROT_READ|syscall.PROT_WRITE|syscall.PROT_EXEC, syscall.MAP_PRIVATE|mmapFlags,
```

Comment:
comment why these flags using a link if necessary

Reply:
@mathetake ready except I don't know what to put here

Reply:
https://github.com/tetratelabs/wazero/pull/155/commits/2f3a70f403e9ea78dbb66a4ea239203d54871f7d Added comments and link!